### PR TITLE
feat/88-Excercise-Group:  Create a exercise  group of a community

### PR DESCRIPTION
**Pull request by phantuananh18**
Testing result:
image

DB changes:
image

### CODE REVIEW

File: src/Services/GTT/shared/GTT.Application/Commands/CreateExGroup.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+using GTT.Application.Repositories;
+using GTT.Application.Requests;
+using GTT.Application.Response;
+using GTT.Domain.Enums;
+using MediatR;
+using System.Net;
+
+namespace GTT.Application.Commands
+{
+    public class CreatExGroup
```
Comment (sdangTMA) [inline src/Services/GTT/shared/GTT.Application/Commands/CreateExGroup.cs:11]: Wrong typo Should change to "Create"

Comment (QuocHuy178) [inline src/Services/GTT/shared/GTT.Application/Commands/CreateExGroup.cs]: Wrong typo should  change to "Create"

**Review by QuocHuy178 — APPROVED** [review]

**Review by ducvp0702 — APPROVED** [review]

**Review by Huyle187 — APPROVED** [review]

**Review by tinophan247 — APPROVED** [review]

File: src/Services/GTT/GTT.Api/GTT.Api.csproj [hunk 2]
```diff
@@ -28,4 +28,7 @@
   <ItemGroup>
     <Using Include="System.Threading.ExecutionContext" Alias="ExecutionContext" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="FunctionHandler\" />
```
Comment (ClearActionConsistent) [inline src/Services/GTT/GTT.Api/GTT.Api.csproj]: just put the api directly in api project to make it consist with other PRs

File: src/Services/GTT/shared/GTT.Application/Response/BaseResponseModel.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+﻿using System.Net;
+
+namespace GTT.Application.Response
+{
+    public class BaseResponseModel
```
Comment (ClearActionConsistent) [inline src/Services/GTT/shared/GTT.Application/Response/BaseResponseModel.cs:5]: each api should has it own response type, change the name BaseResponseModel to something more specific

File: src/Services/GTT/shared/GTT.Domain/Enums/Helper.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿namespace GTT.Domain.Enums
+{
+    public class Helper
```
Comment (ClearActionConsistent) [inline src/Services/GTT/shared/GTT.Domain/Enums/Helper.cs]: This class if contains no dependencies then put it in Application prj

File: src/Services/GTT/shared/GTT.Application/Commands/CreateExGroup.cs [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+                     .NotEmpty().WithMessage("Community is not empty");
+                RuleFor(x => x.data.IsActive)
+                     .NotNull().WithMessage("Active is required")
+                     .NotEmpty().WithMessage("Active is not empty");
+            }
+        }
+
+        internal class Handler : IRequestHandler<Command, BaseResponseModel>
+        {
+            private IUnitOfWork _uow;
```
Comment (ClearActionConsistent) [inline src/Services/GTT/shared/GTT.Application/Commands/CreateExGroup.cs]: the IoW should only meants for transaction, if this command does not require transaction, then we just use repository

File: src/Services/GTT/shared/GTT.Application/Response/BaseResponseModel.cs (hunk 3 above)
Comment (phantuananh18) [reply src/Services/GTT/shared/GTT.Application/Response/BaseResponseModel.cs:5]: The purpose of creating this model is to form a common data return standard, it includes status code, message, and data that change according to their intended use.

**Pull request merged**